src/main/java/frc/robot/Constants.java

```diff
@@ -152,10 +218,39 @@ public static final class ShooterConstants{
         public static final double kFFkA = 0;
 
         //FIXME: create lookup table
-        public static final double [][] shooterLookupTable = {
-            {/* distance to target, shooter speed */}
+        public static final double[][] shooterLookupTable = {
+            {3000,3000,1},
+            {4000,4000,3}
         };
+        public static double[] getVelocitiesFromDistance(double distance) {
+            double[] velocities = new double[2];
 
+            if(distance < shooterLookupTable[0][2]) {
+                velocities[0] = minShooterVelA;
+                velocities[1] = minShooterVelB;
+                return velocities;
+            }
+            if(distance > shooterLookupTable[shooterLookupTable.length-1][2]) {
+                velocities[0] = maxShooterVelA;
+                velocities[1] = maxShooterVelB;
+                return velocities;
+            }
+
+            double[] smaller = new double[3];
+            double[] larger = new double[3];
+
+            for(int i = 0; i < shooterLookupTable.length-2; i++) {
+                if(distance >= shooterLookupTable[i][2] && distance <= shooterLookupTable[i+1][2]) {
+                    smaller = shooterLookupTable[i];
+                    larger = shooterLookupTable[i+1];
+                    break;
+                }
+            }
```

Comment:
same problem as the other lookup. easy fix just change to (length-1) OR change < to <=